test/admin-hpa/features/support/sql_db_helper.rb

```diff
@@ -259,6 +259,12 @@ def self.check_form_details_using_id(check_form_id)
     chk_form_res
   end
 
+  def self.update_check_window_check_end_date_for_development_check_window(check_end_date)
```

Comment:
@sinhaamit1980 consider renaming this long method name. Maybe just 'update_check_window_end_date'